opensrp-giz-malawi/src/main/java/org/smartregister/giz/repository/GizMalawiRepository.java

```diff
@@ -176,9 +181,24 @@ private void runLegacyUpgrades(SQLiteDatabase database) {
         upgradeToVersion13(database);
         upgradeToVersion14(database);
         upgradeToVersion15RemoveUnnecessaryTables(database);
```

Comment:
Please rename this methods correctly. GIZ's DB is at a max of 8 I don't see how you would be upgrading to 15